netcore/Makefile

```diff
@@ -61,11 +73,11 @@ $(ASPNETCORESDK_FILE):
 
 update-corefx: corefx/.stamp-dl-corefx-$(NETCORETESTS_VERSION)
 
-corefx/.stamp-dl-corefx-$(NETCORETESTS_VERSION): corefx-restore.csproj
+corefx/.stamp-dl-corefx-$(NETCORETESTS_VERSION): corefx-restore.csproj init-tools
 	dotnet build corefx-restore.csproj --runtime $(RID) --packages corefx/packages -p:MicrosoftPrivateCoreFxNETCoreAppVersion=$(NETCORETESTS_VERSION) -p:OutputPath=corefx/restore/
```

Comment:
environment is not preserved from init-tools, so this will use dotnet from PATH. Same for all raw calls to `dotnet` - we need to fully qualify those to use the path from `../dotnet/`

Reply:
should be fixed now I guess